0000-template.md

```diff
@@ -29,9 +29,9 @@ Explain the proposal as if it was already included in the language, or in the ke
 - Introducing new named concepts.
 - Explaining the functionality largely through examples.
 - Explaining how CC+ programmers should *think* about the feature and how it should impact on the way they use it. The impact should be explained in as concrete a way as possible.
-- Explaining how the feature fits into use and the relationship between its different parts.
+- Explaining how the feature of the language or the kernel fits into use and the relationship between its different parts.
 - If applicable, provide examples of error messages, depreciation warnings or migration advice.
-- If applicable, describe the differences between teaching this functionality to existing CC+ programmers and to new CC+ programmers.
+- If applicable, describe the differences between teaching this functionality to existing CC+ programmers and to new CC+ programmers. To clarify: the existing programmer will    have to learn a new feature every time there is one.  Whereas the new programmer will learn with the existing features and those added before he arrived in the project. Also, the existing programmer can serve as a referent when the new programmer experiences difficulty with the task.
```

Comment:
Well that have nothing to do with RFCs

Reply:
Um... I didn't understand the initial message. Sorry.